### PR TITLE
Add info on assessment submission

### LINKED ISSUE #47: What's the workflow for grading assignments?
We discussed this briefly before. Should students submit assignments on blackboard, via email, or as pull requests on GitHub? GitHub would be nice just because it is easy to comment on specific lines of a pull request, but would need many repositories so that they cannot see each others work... I guess it works with inline comments in email as well, but it would not be as easy to render it nicely...

### PR DESCRIPTION
**Pull request by joelostblom**
Also added that assessments will be accessible via the homepage. Close #47

### CODE REVIEW

File: index.Rmd [hunk 1]
```diff
@@ -37,7 +37,7 @@ _Please address all general course-related email communication to joel.ostblom@m
 Prof. Martin Krkosek, martin.krkosek@utoronto.ca, 416-978-3839, RW402
 
 ## Course Website and Blackboard
-All course information is accessible [on its own website](https://uoftcoders.github.io/rcourse/) and on [Blackboard](https://portal.utoronto.ca/), including the syllabus and lecture slides. If you have any problem accessing the material, let us know via email right away so we can fix the problem.
+All course information is accessible [on its own website](https://uoftcoders.github.io/rcourse/) and on [Blackboard](https://portal.utoronto.ca/), including the syllabus, assessments and lecture slides. If you have any problem accessing the material, let us know via email right away so we can fix the problem.
```
Comment (lwjohnst86) [inline index.Rmd]: need oxford comma here! `syllabus, assessments, and lecture slides`

**Comment by lwjohnst86** [comment]
Har... har... nice commit message :P

File: index.Rmd [hunk 2]
```diff
@@ -106,4 +106,4 @@ You should be aware of the University of Toronto Code of Behaviour on Academic M
 | Reproducible workflow | Project, Individual | Nov 06     | 8     |
 | Final project work    | Project, Group      | Dec 9 – 20 | 32    |
 
-_There will be a penalty of 5% per day (including week-ends) for late submissions._
+Assessments will be distributed and submitted in the R-markdown format via Blackboard or GitHub. _There will be a penalty of 5% per day (including week-ends) for late submissions._
```
Comment (lwjohnst86) [inline index.Rmd]: I know I'm nit picking here, but it should be `R Markdown`. Sorry, I can sometimes be a perfectionist...

Comment (joelostblom) [reply index.Rmd]: It's good, makes me learn faster! I  actually just corrected this in my assignment after discovering that the official way of writing seems to be `R Markdown` and `R Notebook`, but `RStudio`as one word...

**Pull request merged**